kitsune/questions/config.py

```diff
@@ -140,37 +140,37 @@
                         # TODO: Just use the IA topics for this.
                         # See bug 979397
                         (
-                            "download-and-install",
+                            "install-and-update",
                             {
-                                "name": _lazy("Download, install and migration"),
-                                "topic": "download-and-install",
-                                "tags": ["download-and-install"],
+                                "name": _lazy("Install and Update"),
+                                "topic": "install-and-update",
+                                "tags": ["install-and-update"],
                             },
                         ),
                         (
-                            "privacy-and-security",
+                            "protect-your-privacy",
                             {
                                 "name": _lazy("Privacy and security settings"),
-                                "topic": "privacy-and-security",
-                                "tags": ["privacy-and-security"],
+                                "topic": "protect-your-privacy",
+                                "tags": ["protect-your-privacy"],
                             },
                         ),
                         (
                             "customize",
                             {
-                                "name": _lazy("Customize controls, options and add-ons"),
-                                "topic": "customize",
-                                "tags": ["customize"],
+                                "name": _lazy("Customize settings and Preferences"),
```

Comment:
I don't think "preferences" should be  capitalized?
```suggestion
                                "name": _lazy("Customize settings and preferences"),
```